recipes/zoneinfo/LICENSE

```diff
@@ -0,0 +1,15 @@
+Apache Software License 2.0
+
+Copyright (c) 2020, Paul Ganssle (Google)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
It looks like the `LICENSE` file is included in the source so you can delete this copy and just reference it in the `meta.yaml`

![image](https://user-images.githubusercontent.com/881019/97380746-6c7aeb00-1913-11eb-8267-9dfbca2a5551.png)